index.html

```diff
@@ -1,9 +1,47 @@
 <!DOCTYPE html>
-<hml>
+<html>
 	<head>
 		<title>Introduction to the DOM</title>
-		<link rel="stylesheet" href="./styles.css">
+    <link rel="stylesheet" href="./styles.css">
 	</head>
 	<body>
+    <div class="Header">
+      <script src="components/Dropdown/index.js" async></script>
+      <h1 class="Header__headline">Header</h1>
+      <div class="Dropdown">
+        <h4 class="Dropdown__button">Dropdown</h4>
+        <ul id="myDropdown" class="Dropdown__Menu">
+          <li><a href="https://lambdaschool.com/">Lambda School</a></li>
+          <li><a href="https://www.google.com/">Google</a></li>
+          <li><a href="https://developer.mozilla.org/en-US/">MDN</a></li>
+        </ul> 
+      </div>
+    </div>
+    <div class="Box">
+      <div class="Box__header">
+        <h3 class="Box__headerHeadline">Placeholder</h2>
```

Comment:
You can make thing look something like `Box__header--Headline`. Just a suggestion but if you will have anything nested, the third element will usually come after `--`. 

`Item1__Item2--Item3--Item4 etc`